### PR TITLE
Revise "index_vn"

### LINKED ISSUE #2381: Revise "index_vn"

### PR DESCRIPTION
**Pull request by duythanhvn**
Close #2381

### CODE REVIEW

File: chapter_computer-vision/index_vn.md [hunk 1]
```diff
@@ -24,12 +24,11 @@ Then, we explain how to use style transfer technology to generate images that lo
...
 -->
 
-Chúng tôi đã giới thiệu các mô hình học sâu thường được sử dụng trong lĩnh vực thị giác máy tính trong chương "Mạng nơ-ron tích chập" và đã thực hành một số tác vụ phân loại hình ảnh đơn giản.
-Trong chương này, chúng tôi sẽ giới thiệu các phương pháp tăng cường hình ảnh và các phương pháp tinh chỉnh và áp dụng chúng vào phân loại hình ảnh.
-Sau đó, ta sẽ khám phá các phương pháp nhận diện vật thể khác nhau.
-Sau đó nữa, ta sẽ tìm hiểu cách sử dụng các mạng chập hoàn toàn để thực hiện phân vùng ngữ nghĩa trên hình ảnh.
-Sau đó, chúng tôi giải thích cách sử dụng kỹ thuật chuyển phong cách để tạo ra hình ảnh trông giống như bìa của cuốn sách này.
-Cuối cùng, chúng tôi sẽ thực hiện các bài tập thực hành trên hai bộ dữ liệu thị giác máy tính quan trọng để xem lại nội dung của chương này và các chương trước.
+Ở chương "Mạng Nơ-ron Tích chập", chúng tôi đã giới thiệu các mô hình học sâu thường được sử dụng trong lĩnh vực thị giác máy tính và đã cùng thực hành một số tác vụ phân loại hình ảnh đơn giản.
+Trong chương này, chúng tôi sẽ giới thiệu các phương pháp tăng cường hình ảnh, phương pháp tinh chỉnh và áp dụng chúng vào phân loại hình ảnh.
```
Comment (cuongvng) [inline chapter_computer-vision/index_vn.md:28]:
```suggestion
Trong chương này, chúng tôi sẽ giới thiệu các phương pháp tăng cường hình ảnh (*image augmentation*), phương pháp tinh chỉnh (*fine-tuning*) và áp dụng chúng vào phân loại hình ảnh.
```

File: chapter_computer-vision/index_vn.md [hunk 2]
```diff
@@ -24,12 +24,11 @@ Then, we explain how to use style transfer technology to generate images that lo
...
+Tiếp đến, ta sẽ khám phá các phương pháp nhận diện vật thể khác nhau, cùng tìm hiểu cách sử dụng các mạng tích chập hoàn toàn để thực hiện phân vùng ngữ nghĩa trên hình ảnh.
```
Comment (cuongvng) [inline chapter_computer-vision/index_vn.md:29]:
```suggestion
Tiếp đến, ta sẽ khám phá các phương pháp phát hiện vật thể khác nhau, cùng tìm hiểu cách sử dụng các mạng tích chập đầy đủ để thực hiện phân vùng ngữ nghĩa trên hình ảnh.
```

File: chapter_computer-vision/index_vn.md [hunk 3]
```diff
@@ -24,12 +24,11 @@ Then, we explain how to use style transfer technology to generate images that lo
...
+Sau đó, chúng tôi giải thích cách sử dụng kỹ thuật chuyển phong cách để tạo nên những hình ảnh trông giống như bìa của cuốn sách này.
```
Comment (cuongvng) [inline chapter_computer-vision/index_vn.md:30]:
```suggestion
Sau đó, chúng tôi giải thích cách sử dụng kỹ thuật truyền tải phong cách (*style transfer*) để tạo nên những hình ảnh trông giống như bìa của cuốn sách này.
```

**Review by cuongvng — COMMENTED** [review]
Thank anh!

**Review by cuongvng — APPROVED** [review]

**Review by lkhphuc — APPROVED** [review]

**Pull request merged**